examples/migrations/google-adk/artifacts/adk-live-run/memanto-roundtrip-export/sessions/google-adk-okf-atlascraft-final-20260801_2026-07-06_unknown_summary.md

```
@@ -0,0 +1,64 @@
+
+---
+
+## [2026-07-06 09:00:02] [FACT] Project stack
+- **Memory ID**: `eb9bf1fc-4214-4108-a2c4-ddcc3cdaf0bf`
+- **Confidence**: `0.9`
+- **Status**: `active`
+- **Source**: `tool`
+- **Provenance**: `imported`
+- **Tags**: `app:atlas-release-copilot`, `scope:app`, `source:google-adk`, `type:fact`
+- **Content**:
+> # Project stack
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)